cli/volume.go

```diff
@@ -336,6 +337,13 @@ func (v *VolumeListCommand) runVolumeList(args []string) error {
 		return fmt.Errorf("Conflicting options: --size (or --mountpoint) and -q")
 	}
 
+	//sort.Slice(volumeList.Volumes, func(i, j int) bool {
+	//	return sortorder.NaturalLess(volumeList.Volumes[i].Name, volumeList.Volumes[j].Name)
+	//})
+	sort.Slice(volumeList.Volumes, func(i, j int) bool {
+		return volumeList.Volumes[i].Name < volumeList.Volumes[j].Name
+	})
+
 	display := v.cli.NewTableDisplay()
```

Comment:
could you add an integration test to verify it?

Reply:
Done. PTAL.